Client/src/Components/Profile/PermissionsSection/PermissionsTab.tsx

```diff
@@ -0,0 +1,76 @@
+import * as Yup from 'yup'
+
+import { Box, Button, Grid, Typography } from '@material-ui/core'
+import { FastField, Form, Formik } from 'formik'
+import { fetchAllAdmins, updatePermissions } from '../../../Remote/Endpoints/PermissionsEndpoint'
+import { AdminList } from './AdminList'
+
+import { MuiTextFieldFormik } from '../../Forms/FormikFields'
+import React, { useEffect } from 'react'
+import { classStyles } from '../../../appTheme'
+import { useState } from 'react'
+
+export default function PermissionsTab() {
+
+  const validationSchema = Yup.object().shape({
+    email: Yup.string()
+      .email().required("Email Address")
+  })
+
+  const [adminListState, setAdminListState] = useState([])
+
+  const handleAddAdmin = async (newAdmin: any) => {
```

Comment:
this should have been an interface AddAdminForm. (formValues: AddAdminForm)

Reply:
actually take a look at this comment below:
![image](https://user-images.githubusercontent.com/22376976/107598887-fd3cb480-6bec-11eb-83e0-a66903b92fc4.png)